llvm/lib/ObjCopy/ELF/ELFObject.cpp

```diff
@@ -2987,7 +2987,7 @@ size_t SRECWriter::writeTerminator(uint8_t *Buf, uint8_t Type) {
 
 Expected<size_t>
 SRECWriter::getTotalSize(WritableMemoryBuffer &EmptyBuffer) const {
-  SRECSectionWriterBase LengthCalc(EmptyBuffer, 0);
+  SRECSizeCalculator LengthCalc(EmptyBuffer, 0);
```

Comment:
Nit: it feels a bit off to be mixing the terms "Length" and "Size" for the same thing. I don't mind which you use (although I think "Size" is more common in general), but I'd be consistent.

Reply:
Done